.circleci/config.yml

```diff
@@ -2,10 +2,13 @@
 # See: https://circleci.com/docs/2.0/configuration-reference
 version: 2.1
 
+orbs:
+  browser-tools: circleci/browser-tools@1.4.0
```

Comment:
This is a thing from circle that fixed the webdriver problem: https://circleci.com/developer/orbs/orb/circleci/browser-tools